### PR TITLE
Add: teleinfo sensor documentation

### PR DESCRIPTION
**Pull request by nlamirault**
Signed-off-by: Nicolas Lamirault <nicolas.lamirault@gmail.com>

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14200

## Checklist:

  - [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by c727** [comment]
main PR closed

File: source/_components/sensor.teleinfo.markdown
Comment (DubhAd) [inline source/_components/sensor.teleinfo.markdown]: These should be using `"{{ state_attr('sensor.edf','HCHC') }}"` as this format handles if the sensor isn't available, rather than throwing errors.

File: source/_components/sensor.teleinfo.markdown
Comment (DubhAd) [inline source/_components/sensor.teleinfo.markdown]: And here

File: source/_components/sensor.teleinfo.markdown
Comment (DubhAd) [inline source/_components/sensor.teleinfo.markdown]: And here

File: source/_components/sensor.teleinfo.markdown
Comment (DubhAd) [inline source/_components/sensor.teleinfo.markdown]: And here

File: source/_components/sensor.teleinfo.markdown
Comment (DubhAd) [inline source/_components/sensor.teleinfo.markdown]: And here

File: source/_components/sensor.teleinfo.markdown
Comment (DubhAd) [inline source/_components/sensor.teleinfo.markdown]: And here

File: source/_components/sensor.teleinfo.markdown
Comment (DubhAd) [inline source/_components/sensor.teleinfo.markdown]: And here

File: source/_components/sensor.teleinfo.markdown
Comment (DubhAd) [inline source/_components/sensor.teleinfo.markdown]: Either don't define the unit at all, or define it, don't define it as blank.

File: source/_components/sensor.teleinfo.markdown
Comment (DubhAd) [inline source/_components/sensor.teleinfo.markdown]: And finally here

File: source/_components/sensor.teleinfo.markdown
Comment (DubhAd) [inline source/_components/sensor.teleinfo.markdown]: Please use a configuration block as [detailed here](https://developers.home-assistant.io/docs/en/documentation_standards.html#component-and-platform-pages).

File: source/_components/sensor.teleinfo.markdown
Comment (DubhAd) [inline source/_components/sensor.teleinfo.markdown]: No longer correct, 0.80 I'm guessing?

**Review by DubhAd — DISMISSED** [review]

**Comment by nlamirault** [comment]
Any idea about this error : 

```
10:27:22 AM: failed during stage 'deploying site': Invalid filename 'components/mqtt/#logging.html'. Deployed filenames cannot contain # or ? characters
```

**Comment by frenck** [comment]
@nlamirault You branch is based on an older version of the website. Please rebase your changes onto the latest `next` branch. (Please rebase, do not merge `next` into this branch).

File: source/_components/sensor.teleinfo.markdown
Comment (tjorim) [inline source/_components/sensor.teleinfo.markdown:24]: I think `sensor:` should be on this line.
Increase the indentation of the next lines with 2 spaces as well.

File: source/_components/sensor.teleinfo.markdown
Comment (tjorim) [inline source/_components/sensor.teleinfo.markdown:21]: Since this doesn't contain Jinja2 templates we don't need the `raw`-tags.

File: source/_components/sensor.teleinfo.markdown
Comment (tjorim) [inline source/_components/sensor.teleinfo.markdown:29]: Remove endraw tag

File: source/_components/sensor.teleinfo.markdown
Comment (tjorim) [inline source/_components/sensor.teleinfo.markdown:32]: This is not needed, `{% configuration %}` will include it automatically.

File: source/_components/sensor.teleinfo.markdown
Comment (tjorim) [inline source/_components/sensor.teleinfo.markdown:37]: Not required because of default

**Review by tjorim — CHANGES_REQUESTED** [review]
Some things will need to be adjusted, thanks for the PR!

⚠️ The `ha_release` will need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed as well.

**Pull request closed without merging**
